### ISSUE TITLE
Alternate chains always show ETH as native currency

### ISSUE THREAD

**Issue by nance2uiuc** [torusresearch/torus-embed#292]
**Describe the bug**
I have successfully configured for Binance Smart Chain (test net). In my wallet, everything appears correctly. However, when signing a transaction, prices appear in ETH and converts it to USD which is much higher.

I have successfully signed transactions on the BSC test net without issue. I have also tried to set a field "symbol" in the network config but it does not change anything.

**To Reproduce**
```
await torus.init({
      network: {
        host: "https://data-seed-prebsc-1-s1.binance.org:8545", // mandatory
        chainId: '97', // optional
        networkName: "Binance Smart Chain- Testnet",
        symbol: "BNB"
      },
      showTorusButton: false, // default: true
      enabledVerifiers: {
        facebook: false,
        google: true,
        facebook: false,
        reddit: false,
        twitch: false,
        discord: false,
        twitter: false
      }
    });
```

**Expected behavior**
Fees should appear in BNB and not ETH when a txn is being signed.

**other info**
I also have the account configured for BSC test net from the wallet web page, so that's not the issue.

**Comment by nance2uiuc** [comment]
Any update on this?

**Comment by bitbeckers** [comment]
Hi there, 

having this configurable would be very nice. Our case is that we're moving to Matic and there -at least on Mumbai Testnet- transactions are also displayed in ETH. This could cause friction when onboarding new users.

**Comment by chaitanyapotti** [comment]
This is fixed now and we'll release a new version by the end of this week

**Comment by chaitanyapotti** [comment]
This is fixed in v1.9.10 release
